src/scripts/Controller.js

```diff
@@ -598,7 +601,7 @@ export default class SitemapController {
 											};
 										}
 										if (
-											!sitemap._id.match('^[a-z][a-z0-9_\\$\\(\\)\\+\\-/]+$')
+											!sitemap._id.match('^[a-z][a-z0-9_\\$\\(\\)\\+\\-]+$')
```

Comment:
we can use the same regex pattern here instead of string: `!sitemap._id.match(SITEMAP_ID_REGEXP)`